### ISSUE TITLE
Autodetect GitHub Actions from Environment

### ISSUE THREAD

**Issue by orj** [cpisciotta/xcbeautify#155]
It might be a good idea to auto-detect GitHub Actions from the environment and change the renderer automatically.

Change https://github.com/tuist/xcbeautify/blob/7bef2a55a99f579d4f4d8a5e492191771c32c542/Sources/xcbeautify/Xcbeautify.swift#L27
to:
```swift
var renderer: XcbeautifyLib.Renderer = ProcessInfo.processInfo.environment["GITHUB_ACTIONS"] == "true" ? .gitHubActions : .terminal
```

**Comment by cpisciotta** [comment]
Nice suggestion! I've opened a PR to introduce the auto-detect logic.

**Comment by cpisciotta** [comment]
@orj This is now available with 1.1.0.
